docs/extensions/conventions.md

```diff
@@ -1,5 +1,7 @@
 # General conventions
 
+Gardener determines the time of reconciliation for resources of the API group `extensions.gardener.cloud`. It does that by annotating the respected resource with `gardener.cloud/operation: reconcile`. Extension controllers react to this annotation and start reconciling the resource.
```

Comment:
```suggestion
Gardener dictates the time of reconciliation for resources of the API group `extensions.gardener.cloud`. It does that by annotating the respected resource with `gardener.cloud/operation: reconcile`. Extension controllers react to this annotation and start reconciling the resource. They have to remove this annotation after they have successfully reconciled. In case of errors the annotation should not be removed.
```

Reply:
> They have to remove this annotation after they have successfully reconciled. In case of errors the annotation should not be removed.

As discussed, the controllers remove this annotation right at the beginning of reconciliation. I amended the doc accordingly.